docs/website/docs/getting-started/pytorch.md

```diff
@@ -78,21 +81,33 @@ section of this site for a full list of targets and configurations.
 The generated flatbuffer can now be serialized and stored for another time or
 loaded and executed immediately.
 
+!!! note 
+    The input type `tm_tensor` corresponds to the `linalg-on-tensors`
+    output type of `torch-mlir`.
+    
+!!! note 
+    The conversion to bytecode before passing the module to IREE is needed
+    to cross the border from the Torch-MLIR MLIR CAPI to the IREE MLIR CAPI.
+
 ### Load the VM flatbuffer into IREE
 
-Next, we load the flatbuffer into the IREE runtime. `iree_torch` provides a
-convenience method for loading this flatbuffer from Python:
+Next, we load the flatbuffer into the IREE runtime:
 
 ```python
-invoker = iree_torch.load_vmfb(iree_vmfb, iree_backend)
+config = ireert.Config(driver_name="local-sync")
+ctx = ireert.SystemContext(config=config)
+vm_module = ireert.VmModule.from_flatbuffer(ctx.instance, flatbuffer)
+ctx.add_vm_module(vm_module)
+invoker = ctx.modules.module
```

Comment:
Not as pretty as the `iree_torch` version, but will allow us to remove the `iree_torch` dependence while we figure out how to link to SHARK